composer/trainer/trainer.py

```diff
@@ -1023,7 +1026,14 @@ def saved_checkpoints(self) -> List[Tuple[Timestamp, List[pathlib.Path]]]:
     def fit(self):
         """Train and evaluate the model on the provided data."""
         # Print any exception, so it can be caputred by any callbacks or loggers (e.g. WandB, FileLogger)
-        self._train_loop()
+
+        if self._device == "tpu":
+            import torch_xla.distributed.xla_multiprocessing as xmp            
+            def _mp_fn(index):
+                self._train_loop()
+            xmp.spawn(_mp_fn, args=(), nprocs=8,start_method='fork')
```

Comment:
I'm confused here -- can you elaborate on this change? For distributed training, we use a launch script -- see https://docs.mosaicml.com/en/stable/trainer/distributed_training.html